### ISSUE TITLE
Webhook: How do I assign the events

### ISSUE THREAD

**Issue by freshface** [picqer/moneybird-php-client#170]
Hi

I am creating a webhook:

```
$hook = $moneybird->webhook();
        $hook->url = 'URL';
        $hook->events = ['sales_invoice_send_email','sales_invoice_updated'];
        $hook->save();
```

But how can i define the events. The fillable for webhook does not provide the events.

**Comment by freshface** [comment]
Nobody who can help?

**Comment by stephangroen** [comment]
Your example should work, the docs state you need to pass an array of strings. Are you getting an error?

If you need a list of all available events, that is available here: https://developer.moneybird.com/webhooks/#events.

**Comment by freshface** [comment]
I am not getting an error, just a registered webhook with not events to listen for.
I think this api is not passing the events.

If i do a plain cURL it works.

**Comment by stephangroen** [comment]
You are correct, events didn't get passed. This is fixed in v0.20.1.
